index.js

```diff
@@ -251,7 +256,9 @@ class Flic2 extends EventEmitter {
   stopScan() {
 
     // listen for scanResult
-    this.nativeEvents.removeListener('scanResult', this.onScanResultFunction);
+    if (this.scanResultListener) {
```

Comment:
This guard condition is added to protect against any operations being done on the listener when it has not yet been initialised.